index.js

```diff
@@ -16,14 +19,21 @@ const appPort = process.env.PORT || 3001
 
 const bodyParser = require('body-parser')
 
-msLogger.setLogLevel(process.env.LOG_LEVEL || msLogger.LOG_LEVELS.DEBUG)
+logger.setLogLevel(process.env.LOG_LEVEL || logger.LOG_LEVELS.DEBUG)
 
 const app = express()
 app.use(bodyParser.json())
 app.use(msCorrelation.correlationMiddleware)
+app.use(msLogger.middleware)
```

Comment:
The logger middleware if called `loggerMiddleware`, and not just `middleware`

Reply:
That was working....

Reply:
You didn't run `yarn upgrade`